fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -84,20 +82,16 @@
 
     @Override
     public void initialize(final NamespaceRegistry registry,
-                           final NodeTypeManager nodeTypeManager) throws IOException {
-        try {
```

Comment:
As you can see from the previous commit on this file, we want to stop the jcr exception `RepositoryException` from leaking out of this class:
https://github.com/acoburn/fcrepo4/commit/de826b28a2d8c0fb81ea4cc9db0a6bab5f90643b#diff-172c1a74effcaea58305292a8256a1b9